File: TFMVVM-Example/TFMVVM-Example/ViewModel/SimpleRowViewModel.m
//
//  SimpleRowViewModel.m
//  TFMVVM-Example
//
//  Created by Krzysztof on 08/08/2015.
//  Copyright (c) 2015 Trifork. All rights reserved.
//

#import "SimpleRowViewModel.h"

@implementation SimpleRowViewModel

+ (instancetype)withColor:(UIColor *)color name:(NSString *)name
{
    SimpleRowViewModel * vm = [[[self class] alloc] initWithModel:nil];
    vm.bgdColor = color;
    vm.name = name;
    return vm;
}

@end

File: TFMVVM-Example/TFMVVM-Example/ViewModel/SimpleRowViewModel.swift
//
//  SimpleRowViewModel.m
//  TFMVVM-Example
//
//  Created by Krzysztof on 08/08/2015.
//  Copyright (c) 2015 Trifork. All rights reserved.
//

import UIKit

class SimpleRowViewModel: TFBaseViewModel {
    var bgdColor: UIColor?
    var name: String?
    
    init(color: UIColor, name: String) {
        self.name = name
        self.bgdColor = color
        super.init(model:nil)
    }
}